src/Network/Receive.pm

```diff
@@ -1088,13 +1100,22 @@ sub account_server_info {
 		('-'x34) . "\n", 'connection';
 
 	@servers = @{$args->{servers}};
-
-	my $msg = center(T(" Servers "), 53, '-') ."\n" .
-			T("#   Name                  Users  IP              Port\n");
-	for (my $num = 0; $num < @servers; $num++) {
-		$msg .= swrite(
-			"@<< @<<<<<<<<<<<<<<<<<<<< @<<<<< @<<<<<<<<<<<<<< @<<<<<",
-			[$num, $servers[$num]{name}, $servers[$num]{users}, $servers[$num]{ip}, $servers[$num]{port}]);
+	my $msg = center(T(" Servers "), 53, '-') ."\n";
+	if ($masterServer->{serverType} eq "tRO") {
```

Comment:
I think dont need add new one ? .

`$servers[$num]{sid} ? $servers[$num]{sid} : '';
`

Reply:
Only tRO sent server select sent SID but not user count.
we need to remove user column in case of tRO.